packages/trackerless-network/test/integration/stream-without-default-entrypoints.test.ts

```diff
@@ -0,0 +1,132 @@
+import { NetworkNode } from '../../src/NetworkNode'
+import { range } from 'lodash'
+import { NodeType, PeerDescriptor, PeerID, Simulator, SimulatorTransport, LatencyType } from '@streamr/dht'
+import {
+    MessageID,
+    MessageRef,
+    StreamMessage,
+    StreamMessageType,
+    StreamPartIDUtils,
+    toStreamID
+} from '@streamr/protocol'
+import { EthereumAddress, waitForCondition } from '@streamr/utils'
+import { streamPartIdToDataKey } from '../../src/logic/StreamEntryPointDiscovery'
+
+describe('stream without default entrypoints', () => {
+
+    let entrypoint: NetworkNode
+    let nodes: NetworkNode[]
+    let numOfReceivedMessages: number
+    const entryPointPeerDescriptor: PeerDescriptor = {
+        kademliaId: new Uint8Array([1, 2, 3]),
+        nodeName: 'entrypoint',
+        type: NodeType.NODEJS
+    }
+
+    const STREAM_ID = StreamPartIDUtils.parse('test#0')
+    const streamMessage = new StreamMessage({
```

Comment:
Could use helper method for creating a test message so that we know if there are any message values which are relevant to this test?